types/lib/Server.d.ts

```diff
@@ -716,9 +717,9 @@ declare class Server {
           description: string;
           multiple: boolean;
           path: string;
-          type: string;
+          type: string /** @type {WebSocketURL} */;
```

Comment:
Regenerating the Server types introduces a lot unrelated comments in the type definitions, but running `build:types` on master does not create any diff.

Reply:
Hmm, types changed here since you added [` * @param {string} host`](https://github.com/webpack/webpack-dev-server/pull/4385/files#diff-b706bbadb3d2315d99678f05e1f4bfd78a4bb4a7a6c65f3e81f1386fee3c883a)

Reply:
It is bug in ts, reported about it, but still not fixed